app/styles/_navbar-vertical.less

```diff
@@ -56,11 +56,15 @@
     @media(min-width: @screen-sm-min) {
       margin-right: @navbar-header-right-margin;
     }
-    .badge {
+    .nav-item-iconic .badge {
+      background-color: @color-pf-blue-300;
+      border: 2px solid @navbar-os-bg-color;
+      height: 12px;
```

Comment:
I don't think this should be straying from the Patternfly design, @openshift/team-ux-review ?

Reply:
@sg00dwin What is the before and after on this change?

If @openshift/team-ux-review agrees it's better, we could look at contributing it upstream in Patternfly.

Reply:
@sg00dwin again, why are we doing this?  Was this suggested by @jhaines?  

If there's a good reason, let's get this into PF.  LOTS of products are using the notification drawer, we want this to be consistent if it's indeed something that improves the component.

Reply:
discussed with rhamilto, done to improve identification of unread notification. 
Will revert

before 
<img width="43" alt="screen shot 2017-09-01 at 2 42 11 pm" src="https://user-images.githubusercontent.com/1874151/29983446-dd1add16-8f23-11e7-922b-18133ad5db03.png">

after
<img width="32" alt="screen shot 2017-09-01 at 2 41 18 pm" src="https://user-images.githubusercontent.com/1874151/29983452-e2e5e7cc-8f23-11e7-8c90-a0c093a09881.png">

Reply:
@serenamarie125 Opinion? I think the badget stands out better with the black around it. Let us know, and we can add submit this in Patternfly.

Before:

![openshift web console 2017-09-01 14-48-41](https://user-images.githubusercontent.com/1167259/29983598-ad9c16f8-8f24-11e7-99a4-12a2ce25389e.png)

After:

![openshift web console 2017-09-01 14-47-54](https://user-images.githubusercontent.com/1167259/29983572-8f234674-8f24-11e7-8be1-5a36b2842a2f.png)

Reply:
I think it's worth noting the inspiration for the border around the badge came from Github.

Reply:
again, imo @sg00dwin @rhamilto I think your proposal looks great, and i did immediately recognize that it was consistent with GitHub.  

Thoughts on putting up a PR in Core PF for this ?